provision/core/env.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+#
+# core/env.sh
+
+setup_vvv_env() {
+  # fix no tty warnings in provisioner logs
+  sudo sed -i '/tty/!s/mesg n/tty -s \\&\\& mesg n/' /root/.profile
+
+  # add homebin to secure_path setting for sudo, clean first and then append at the end
+  sed -i -E \
+    -e "s|:/srv/config/homebin||" \
+    -e "s|/srv/config/homebin:||" \
+    -e "s|(.*Defaults.*secure_path.*?\".*?)(\")|\1:/srv/config/homebin\2|" \
+    /etc/sudoers
+
+  # add homebin to the default environment, clean first and then append at the end
+  sed -i -E \
+    -e "s|:/srv/config/homebin||" \
+    -e "s|/srv/config/homebin:||" \
+    -e "s|(.*PATH.*?\".*?)(\")|\1:/srv/config/homebin\2|" \
+    /etc/environment
+}
+
+cleanup_terminal_splash() {
+  # Dastardly Ubuntu tries to be helpful and suggest users update packages
+  # themselves, but this can break things
+  if [[ -f /etc/update-motd.d/00-header ]]; then
```

Comment:
Wondering if we can do something to remove all of them and simplify this

Reply:
we don't need to cleanup MOTD, but it does mean when you SSH in you'd get a tonne of ubuntu advertisements and random stuff about cloud services etc. So I'd prefer to keep it

Reply:
Me too but like remove all the files from that folder can be an idea?